client/state/data-layer/wpcom/sites/comments/likes/mine/delete/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import {
+	COMMENTS_LIKE,
+	COMMENTS_UNLIKE,
+} from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { local } from 'state/data-layer/utils';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+
+export const unlikeComment = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'POST',
+		apiVersion: '1.1',
+		path: `/sites/${ action.siteId }/comments/${ action.commentId }/likes/mine/delete`
+	}, action ) );
+};
+
+export const updateCommentLikes = ( { dispatch }, { siteId, postId, commentId }, next, { i_like, like_count } ) => dispatch( local( {
+	type: COMMENTS_LIKE,
+	siteId,
+	postId,
+	commentId,
+	i_like,
```

Comment:
are we going to update the reducer here because it looks like it's not watching for `i_like`, only passing `true` for that value. further, if we wanted to clear this up a bit we could choose the action based on `i_like`…

```js
{
	type: i_like ? COMMENTS_LIKE : COMMENS_UNLIKE,
	siteId,
	postId,
	commentId,
	like_count,
}
```